eigen/examples/svd/main/idf_component.yml

```
@@ -0,0 +1,10 @@
+## IDF Component Manager Manifest File
+dependencies:
+  espressif/eigen: 
+    version: "^3.4.0"
+    # This line define the local path of the eigen component because this 
+    # example is part of the eigen component. This line is optional.
+    override_path: "../../.."
+  ## Required IDF version
+  idf:
+    version: ">=4.1.0"
```

Eigen itself depends on IDF v4.3 or later here:
https://github.com/espressif/idf-extra-components/blob/a4049b54515f14e5066611f8087f23bd9a47d11b/eigen/idf_component.yml#L6
so I suggest making this consistent
```suggestion
    version: ">=4.3.0"
```